src/engine/Operation.h

```diff
@@ -151,7 +154,18 @@ class Operation {
 
   // Create and return the runtime information wrt the size and cost estimates
   // without actually executing the query.
-  virtual void createRuntimeInfoFromEstimates() final;
+  virtual void createRuntimeInfoFromEstimates(
+      std::shared_ptr<RuntimeInformation> runtimeInformation,
+      std::shared_ptr<RuntimeInformation> rootRuntimeInformation) final;
```

Comment:
Please specify the semantics of the arguments, they are rather important I guess.